src/main/java/module/siadap/domain/SiadapRootModule.java

```diff
@@ -107,6 +108,9 @@ private SiadapRootModule() {
         super();
         setBennu(Bennu.getInstance());
         setNumber(0);
+        final User adminUser = User.findByUsername("ist23487");
```

Comment:
We cannot have this code in the Siadap Module code base.
User ist23487 is specific to IST.